packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/index.test.tsx

```diff
@@ -4,29 +4,67 @@
 
 import React from 'react';
 import '@testing-library/jest-dom/extend-expect';
-import { render, screen } from '@testing-library/react';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
 // import { getFtlBundle, testAllL10n } from 'fxa-react/lib/test-utils';
 // import { FluentBundle } from '@fluent/bundle';
-import { usePageViewEvent } from '../../../lib/metrics';
+import { logViewEvent, usePageViewEvent } from '../../../lib/metrics';
 import ConfirmSignupCode, { viewName } from '.';
-import { MOCK_ACCOUNT } from '../../../models/mocks';
 import { REACT_ENTRYPOINT } from '../../../constants';
+import { Account, AppContext } from '../../../models';
+import { mockAppContext, MOCK_ACCOUNT } from '../../../models/mocks';
+import { LocationProvider } from '@reach/router';
 
 jest.mock('../../../lib/metrics', () => ({
   usePageViewEvent: jest.fn(),
   logViewEvent: jest.fn(),
 }));
 
-describe('PageSigninTokenCode', () => {
+let account: Account;
```

Comment:
Super nit, if you don't need to reference `account` in the global test scope you could move it under the first `describe`.